inputfiles/idl/SVG - Basic Data Types and Interfaces.widl

```diff
@@ -180,6 +180,14 @@ interface SVGAnimatedLengthList {
   [SameObject] readonly attribute SVGLengthList animVal;
 };
 
+[Exposed=Window]
+interface SVGAnimationElement : SVGElement {
+  readonly attribute SVGElement targetElement;
+  float getStartTime();
+  float getCurrentTime();
+  float getSimpleDuration();
+};
```

Comment:
Thanks for jumping in.
Is this a change in the widl from upstream? 
I suspect no, so the [correct place](https://github.com/Microsoft/TSJS-lib-generator#input-files) would be the addTypes.json